pandas/core/frame.py

```diff
@@ -5943,7 +5945,7 @@ def swaplevel(self, i=-2, j=-1, axis=0) -> DataFrame:
             result.columns = result.columns.swaplevel(i, j)
         return result
 
-    def reorder_levels(self, order, axis=0) -> DataFrame:
+    def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
```

Comment:
This has to be `Sequence` not `List` because with `List` mypy complains
```
pandas/core/generic.py:8689: error: Argument 1 to "reorder_levels" of "DataFrame" has incompatible type "List[int]"; expected "List[Union[str, int]]"  [arg-type]
pandas/core/generic.py:8689: note: "List" is invariant -- see http://mypy.readthedocs.io/en/latest/common_issues.html#variance
pandas/core/generic.py:8689: note: Consider using "Sequence" instead, which is covariant
pandas/core/generic.py:8691: error: Argument 1 to "reorder_levels" of "DataFrame" has incompatible type "List[int]"; expected "List[Union[str, int]]"  [arg-type]
pandas/core/generic.py:8691: note: "List" is invariant -- see http://mypy.readthedocs.io/en/latest/common_issues.html#variance
pandas/core/generic.py:8691: note: Consider using "Sequence" instead, which is covariant
```